src/main/resources/templates/db_scripts/invoices_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE INDEX IF NOT EXISTS invoices_invoice_date_sort ON ${myuniversity}_${mymodule}.invoices
+  (left(lower(f_unaccent(invoices.jsonb->>'invoiceDate')),600), lower(f_unaccent(invoices.jsonb->>'invoiceDate')));
+
+CREATE INDEX IF NOT EXISTS invoices_vendor_invoice_no_sort ON ${myuniversity}_${mymodule}.invoices
```

Comment:
is it ok that we name this index as no_sort?

Reply:
Is stands for vendor invoice number